src/Model/Contact.php

```diff
@@ -1354,4 +1354,113 @@ public static function updateSslPolicy($contact, $new_policy)
 
 		return $contact;
 	}
+
+	public static function newFollower($importer, $contact, $datarray, $item, $sharing = false) {
```

Comment:
Please rename this function to `Contact::addRelationship`. In general, all functions that alter the database should be named after a verb.